move/axelar_gateway/sources/types/message_status.move

```diff
@@ -17,7 +17,7 @@ public enum MessageStatus has copy, drop, store {
 // -----------------
 // Package Functions
 // -----------------
-public (package) fun approved(hash: Bytes32): MessageStatus {
+public(package) fun approved(hash: Bytes32): MessageStatus {
```

Comment:
the function below requires this as well. do a find and replace all in the editor?